src/apps/fuzzers/README.md

```diff
@@ -13,56 +13,57 @@ The public API of H3 is covered in the following fuzzers:
 
 | Function | File or status
 | -------- | --------------
-| latLngToCell | [fuzzerLatLngToCell](./fuzzerLatLngToCell.c)
-| cellToLatLng |  [fuzzerCellToLatLng](./fuzzerCellToLatLng.c)
+| areNeighborCells | [fuzzerDirectedEdge](./fuzzerDirectedEdge.c)
```

Comment:
for my curiosity - was this an editor feature you used to sort this? or did you manually alphabetize?

Reply:
Simple `:sort` in vim after highlighting the lines I wanted to sort. :)

Reply:
I figured it must be some *vim* thing :)